tempel.el

```diff
@@ -254,7 +254,7 @@ BEG and END are the boundaries of the modification."
         (save-excursion
           (goto-char (overlay-start ov))
           (let (x)
-            (setq x (or (and (setq x (overlay-get ov 'tempel--form)) (eval x (cdr st)))
+            (setq x (or (and (setq x (overlay-get ov 'tempel--form)) (or (eval x (cdr st)) ""))
                         (and (setq x (overlay-get ov 'tempel--name)) (alist-get x (cdr st)))))
```

Comment:
This change cannot be correct since the other expression with the `alist-get` in the outer `or` won't be executed.

Reply:
It would be executed if `(overlay-get ov 'tempel--form)` is nil, right? And my understanding is that this is the right condition for it to be executed.

Reply:
Ah, you're probably right. We won't have both tempel--form and tempel--name? It is a while that I've hacked on Tempel...

Reply:
No, forms don't have names and named fields don't have forms.